policyengine_us/parameters/gov/states/ga/tax/income/credits/surplus_tax_rebate/amount.yaml

```diff
@@ -0,0 +1,27 @@
+description: Georgia provides a surplus tax rebate of this amount, based on filing status.
+
+metadata:
+  unit: currency-USD
+  period: year
+  breakdown:
+    - filing_status
+  label: Georgia surplus tax rebate amount
+  reference:
+    - title: 2024 CODE OF GEORGIA, Title 48, Article 2, Section 48-7-20.2 (b)(1)(B)
+      href: https://law.justia.com/codes/georgia/title-48/chapter-7/article-2/section-48-7-20-2/
+
+JOINT:
+  2022-01-01: 500
+  2023-01-01: 0
```

Comment:
I dont think this reflects the legal code. The value is not set to 0 but we explicitly remove the program from the computation tree in `non_refundable.yaml`